src/organisation/containers/update-pba-numbers/update-pba-numbers.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { OrganisationDetails } from '../../../models/organisation.model';
+import * as fromStore from '../../store';
+
+@Component({
+  selector: 'app-prd-update-pba-numbers-component',
+  templateUrl: './update-pba-numbers.component.html',
+})
+export class UpdatePbaNumbersComponent implements OnInit {
+
+  public readonly title = 'Add or remove PBA accounts';
+
+  public organisationPaymentAccount: string[];
+
+  constructor(private orgStore: Store<fromStore.OrganisationState>) { }
+
+  ngOnInit() {
```

Comment:
ts lint is warning needs a public modifier

Reply:
Okay... I will add it, but reluctantly. I really disagree with adding the public accessor keyword to the implemented methods such as ngOnInit(). That goes against standard angular coding practices.